code/ATMOSPHERICS/datum_pipeline.dm

```diff
@@ -25,6 +25,11 @@
 	for(var/obj/machinery/atmospherics/pipe/M in members) //Edges are a subset of members
 		M.parent = null
 
+/datum/pipeline/resetVariables()
+	..("members", "edges")

```

Comment:
All parent calls for resetVariables should return args alongside their string arguments, to ensure inheritance works properly.